.pipelines/templates/test-docs.yml

```diff
@@ -0,0 +1,76 @@
+trigger: none

```

Comment:
This is still not attached to the PR pipeline, right?

Reply:
nope